pxr/imaging/plugin/hdRpr/rprApiAov.cpp

```diff
@@ -141,90 +216,232 @@ bool HdRprApiAov::GetData(void* dstBuffer, size_t dstBufferSize) {
     return false;
 }
 
-void HdRprApiAov::Resize(int width, int height, HdFormat format) {
+void
+HdRprApiAov::Resize(int width, int height, HdFormat format, float renderResolution)
+{
+    if (m_width != width || m_height != height || m_renderResolution != renderResolution) {
+        m_width = width;
+        m_height = height;
+        m_renderResolution = renderResolution;
+        
+        // Mark size dirty here because when upscaler gets enabled, RIF size should be updated while RPR not
```

Comment:
Isn't we mark it dirty because the size, well, changed? This comment seems weird because it contains implementation details of the derived class.

Reply:
Comment was initially created in different code path, where only renderResolution was changed. Removed